Topcoder/FoldingPaper2.cc

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include <vector>
+#include <cmath>
+#include <utility>
+#define oo 1e8
+using namespace std;
+class FoldingPaper2 {
+	public:
+	vector <pair <int,int> > findDiv(int n) {
```

Comment:
Don't use spaces before, after, and inside `<` and `>`.
Use `C++11` to allow `vector<vector<int>>` code.
See https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace . Scroll down to **Templates and Casts**.